### PR TITLE
MAT-9402 change to use design system DateField

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-9402](https://jira.cms.gov/browse/MAT-9402)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx [hunk 1]
```diff
@@ -396,96 +394,78 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
...
+            formik.touched.measurementPeriodStart &&
+            Boolean(formik.errors.measurementPeriodStart)
+          }
+          helperText={formikErrorHandler("measurementPeriodStart", true)}
+          value={formik.values.measurementPeriodStart}
+          textFieldSx={{ width: "100%" }}
+          {...formik.getFieldProps("measurementPeriodStart")}
+          InputProps={{
+            "data-testid": "measurement-period-start-input",
+            "aria-describedby": "create-measure-period-start-helper-text",
```
Comment (adongare) [inline src/components/NewMeasure/CreateNewMeasureDialog.tsx]: maybe `measurement-period-start-helper-text`?

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx [hunk 2]
```diff
@@ -396,96 +394,78 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
...
+            <>
+              Measurement Period - End Date
+              <span style={{ color: "rgb(174, 28, 28)", marginLeft: 2 }}>
+                *
+              </span>
+            </>
+          }
+          InputProps={{
+            "data-testid": "measurement-period-end-input",
+            "aria-describedby": "create-measure-period-end-helper-text",
```
Comment (adongare) [inline src/components/NewMeasure/CreateNewMeasureDialog.tsx]: and `measurement-period-end-helper-text`?

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/NewMeasure/CreateNewMeasureDialog.tsx]: @adongare done thanks!

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/NewMeasure/CreateNewMeasureDialog.tsx]: @adongare done.

**Review by adongare — APPROVED** [review]
